Editor/LinkXmlInstaller.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) xRealityLabs. All rights reserved.
+
+using System.IO;
+using UnityEditor;
+using UnityEditor.Build;
+using UnityEditor.Build.Reporting;
+using UnityEditor.UnityLinker;
+
+namespace xRealityLabs.XRFoundation.Editor
+{
+    public class LinkXmlInstaller : IUnityLinkerProcessor
```

Comment:
This doesn't look right.

Reply:
Could you elaborate, as it's the standard Unity template to allow a project consuming this package to instal the Link.xml file contained within.
The same pattern is used in the ASA package, to ensure Unity includes the provided Link.xml file to prevent Unity stripping parts of the toolkit.  Else in some circumstances, you cannot build as the IL2CPP stripper removes code it "thinks" is not needed, which is

Reply:
Oh wait, I see the issue

Reply:
Updated license and namespace